apps/hash-ai-worker-ts/src/activities/graph.ts

```diff
@@ -35,11 +36,15 @@ export const createGraphActivities = ({
       actorId: AccountId;
     };
     embeddings: EntityEmbedding[];
+    updatedAtTransactionTime: Timestamp;
+    updatedAtDecisionTime: Timestamp;
   }): Promise<void> {
     await graphApiClient
       .updateEntityEmbeddings(params.authentication.actorId, {
         embeddings: params.embeddings,
-        reset: true,
+        reset: false,
```

Comment:
In what circumstance would I want to pass `reset: true`? Is it something we actually want users to be able to do?

Reply:
Actually, that was by accident (for some local testing) and this should be `reset: true`. If you would remove a property from an entity the embedding would not be removed. As we neither have partial updates, nor we detect changes to the previous entity, all embeddings for an entity has to be generated anyway.

```suggestion
        reset: true,
```

Reply:
Got it, thank you. In what case would I pass `result: false` then?

Reply:
When it's possible to detect that no property was removed, it's not needed to reset the properties. Possibly we can be even smarter when we have the ability to detect changes. We can also remove it for the time being.

Reply:
It's fine to leave, since we will only be calling this in one place.